src/components/Form.jsx

```diff
@@ -0,0 +1,96 @@
+
+
+import { useFormik } from 'formik';
+import { basicSchema } from '../schemas';
+import './Form.css'
+import styled from 'styled-components'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faClipboardCheck } from '@fortawesome/free-solid-svg-icons';
+import Show from './Show';
+import Navbar from './Navbar';
+import { useState } from 'react';
```

Comment:
lets order the imports:
- first add react libraries (react, react-redux, react-router)
- other extrenal libraries, formik, yup ..
- components
- other files and assets
- styles